src/avram/tasks/db/reset.cr

```diff
@@ -3,6 +3,9 @@ require "colorize"
 class Db::Reset < BaseTask
   summary "Drop, recreate, and run migrations."
 
+  def initialize(@quiet : Bool = false)
```

Comment:
This instance variable will need to be passed in to the `new()` methods in the `run_task` method below so those will pick up the option.